lms/templates/footer.html

```diff
@@ -10,12 +10,14 @@
 <% footer = get_footer(is_secure=is_secure) %>
 <%namespace name='static' file='static_content.html'/>
 
-<footer class="border-top"">
+<footer class="border-top">
     <div class="container">
         <div class="row">
             <div class="col-lg-4 mr-auto">
                 <h4 class="mt-5 mb-3">Navoica.pl</h4>
                 <p align="justify">Polska platforma z kursami typu MOOC, dostępnymi w&nbsp;języku polskim, bez opłat i dla każdego zarejestrowanego użytkownika.</p>
+                <p class="display-none">czytaj: Nawojka lub Nawoika</p>
```

Comment:
Kiedyś google czepiało się i to bardzo jeżeli jakieś elementy na stronie były ukryte i napisane tylko pod roboty indexujące